update-common/src/artifacts/update_plan.rs

```diff
@@ -895,11 +958,11 @@ mod tests {
         tarball: Bytes,
     }
 
-    fn make_random_rot_image() -> RandomRotImage {
+    fn make_random_rot_image(sign: &str, board: &str) -> RandomRotImage {
         use tufaceous_lib::CompositeRotArchiveBuilder;
 
-        let archive_a = make_random_bytes();
-        let archive_b = make_random_bytes();
+        let archive_a = make_fake_rot_image(sign, board);
+        let archive_b = make_fake_rot_image(sign, board);
```

Comment:
haha I was wondering when this would become less random, thanks for doing this :)